tests/container_disk_test.go

```diff
@@ -62,8 +62,13 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	})
 
 	DescribeTable("should", func(image string, policy k8sv1.PullPolicy, expectedPolicy k8sv1.PullPolicy) {
-		vmi := tests.NewRandomVMIWithEphemeralDisk(image)
+		vmi := libvmi.New(libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+			libvmi.WithNetwork(v1.DefaultPodNetwork()),
+			libvmifact.WithCirrosRequestedMemory(),
+			libvmi.WithContainerDisk("disk0", image))
```

Comment:
When we touch something like this, we see how odd it is.

First, the network settings are redundant.
Second, I see no reason to choose Cirros memory, as it is not a Cirros image... or at least I do not see it is.

I also think the formatting you used here can be improved to
```
libvmi.New(
    a(),
    be(),
    ...
)
```
But that is a nit.

This VMI creation could be simplified to:
```go
libvmi.New(
    libvmi.With ResourceMemory("256Mi"),
    libvmi.WithContainerDisk("disk0", image),
)
```

I suspect the memory does not even matter, it could just use `libvmi.NewGuestless` and you will not need the memory thing option at all. Ending up with a simple:
```go
libvmi.NewGuestless(libvmi.WithContainerDisk("disk0", image))
```

Reply:
let's try